.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## Summary of Changes
+
+## Unit Tests
+
+## Documentation Updates
+
```

Comment:
1. We should also do upgrade assessment from all previous supported release.
2. We should do impact assessment on OCP vs K8

Reply:
Let us add "Upgrade considerations". Let us not make the PR template heavy by adding specific like OCP vs K8 difference which in most cases wont be there.